agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -1094,7 +1094,30 @@ private static void validateRegex(String value, ProcessorType processorType) {
 
   public static class ProcessorAttribute {
     public String key;
-    public String value;
+    public Object value;
+    public AttributeType type;
```

Comment:
I think we need to default to STRING for backwards compatibility

Reply:
default to String type.